llama_cpp/server/app.py

```diff
@@ -882,19 +653,16 @@ class ModelList(TypedDict):
 
 @router.get("/v1/models")
 async def get_models(
-    settings: Settings = Depends(get_settings),
+    llama: Llama = Depends(get_llama),
 ) -> ModelList:
-    assert llama is not None
     return {
         "object": "list",
         "data": [
             {
-                "id": settings.model_alias
-                if settings.model_alias is not None
-                else llama.model_path,
+                "id": model,
```

Comment:
This should probably have similar logic to the left side, id should only dump the "name" of the model which can be used for selecting the model in completion and chat_completion requests.